### PR TITLE
Update SortDescription.xml

### PR DESCRIPTION
**Pull request by Omcsesz**
## Summary

I added that if someone uses empty string as the value for parameter propertyName.

### CODE REVIEW

File: xml/System.ComponentModel/SortDescription.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
         <Parameter Name="direction" Type="System.ComponentModel.ListSortDirection" />
       </Parameters>
       <Docs>
-        <param name="propertyName">The name of the property to sort the list by.</param>
+        <param name="propertyName">The name of the property to sort the list by. Should be set to "" if the collection to be sorted contains strings, integers, etc..</param>
```
Comment (tarekgh) [inline xml/System.ComponentModel/SortDescription.xml:42]: > Should be set to "" if the collection to be sorted contains strings, integers, etc.

Should be set to empty string if the collection to be sorted contains types like string or numeric types.

**Review by tarekgh — APPROVED** [review]
I am not the best at rewording it but @gewarren may help in better rewording :-)

File: xml/System.ComponentModel/SortDescription.xml (hunk 1 above)
Comment (Omcsesz) [reply xml/System.ComponentModel/SortDescription.xml:42]: Agreed.

Comment (gewarren) [reply xml/System.ComponentModel/SortDescription.xml:42]:
```suggestion
        <param name="propertyName">The name of the property to sort the list by. If the collection contains strings or value types, set this parameter to the empty string (`""`).</param>
```

**Review by maryamariyan — APPROVED** [review]
@gewarren is it OK to include `""` in the docs as provided in this PR?

**Comment by gewarren** [comment]
> @gewarren is it OK to include `""` in the docs as provided in this PR?

I had left a suggestion to code fence the quotes (and also to reword it slightly) but the conversation was somehow resolved.

https://github.com/dotnet/dotnet-api-docs/pull/6604/files#r1002113425

**Comment by tarekgh** [comment]
Thanks @gewarren, I have applied your suggestion and we can merge after building.

**Pull request merged**